routes/hotels.js

```diff
@@ -1,37 +1,38 @@
 const mongoose = require('mongoose')
 const express = require('express')
 const { Hotel } = require('../models/hotel')
+const ApiError = require('../helpers/apiError')
 const router = express.Router()
 
 router.get('/', async (req, res) => {
   const hotels = await Hotel.find()
 
-  res.status(200).send(hotels)
+  res.status(200).json({ hotels: hotels })
```

Comment:
Why.. you are making front devs job more difficult.
Send(hotels) 

Sending objects makes sense in error 
Const error = err.message 
Where we want to exclude message

Reply:
I was told to make everything json, but no problem 😄